internal/service/todo_service.go

```diff
@@ -0,0 +1,113 @@
+package todo
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	models "github.com/daniyalumer/todo-list-go-chi/internal/models"
+)
+
+func CreateTodo(description string, userid int) (string, models.Todo) {
```

Comment:
Do not return success or failure string from service layer. Use error messages in case of failure and return success messages in controller layer in case of success